test

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+#git fetch origin
+
+function diff_files () {
+    git diff origin/master --name-only
+}
+
+function search () {
+    git diff origin/master --name-only | xargs python3 stats.py --build --files
+}
+
+cmd="$1"
+
+if [ "$cmd" = "--staged-only" ] || [ "$cmd" = "-so" ]; then
```

Comment:
`--staged-only` means the `stage` files before commit them, so this doesn't make sense on running on travis.

This type of usage is only designed to use on local changes. 

I suggest you to change this flag for now as `--commited-only` or something more clear, `--from-master`, `--master`. But `--staged-only` is [another thing](https://softwareengineering.stackexchange.com/questions/119782/what-does-stage-mean-in-git), ok?

Reply:
> But --staged-only is another thing, ok? 

Yes , I will rename almost everything to make things clearer. Btw how many commands to we need at /test file : --unstanged, --master maybe ?

Reply:
I think only `--staged` and `--master` is enough. And later `--sync` and `--async` to select the desired implementation, but this can be in another PR since we don't have yet a proper implementation of this on `stats.exs`.